app/protected/route.js

```diff
@@ -3,13 +3,12 @@ import Ember from 'ember';
 export default Ember.Route.extend({
 
   model(){
-
       return Ember.RSVP.hash({
         jobs: this.store.query('job', {
-          orderBy: "isActive",
-          equalTo: true
+          orderBy: "assignedTo",
```

Comment:
Seems like a decent solution. Is prefer not to pull unnecessary data, but of the options we have this Does the least damage.